pkg/moov/terminal_application_models.go

```diff
@@ -0,0 +1,41 @@
+package moov
+
+type TerminalApplicationStatus string
+
+// List of TerminalApplicationStatus
+const (
+	TerminalApplicationStatus_Pending  TerminalApplicationStatus = "pending"
+	TerminalApplicationStatus_Enabled  TerminalApplicationStatus = "enabled"
+	TerminalApplicationStatus_Disabled TerminalApplicationStatus = "disabled"
+)
+
+type TerminalApplicationPlatform string
+
+// List of TerminalApplicationPlatform
+const (
+	TerminalApplicationPlatform_Android TerminalApplicationPlatform = "android"
+	TerminalApplicationPlatform_IOS     TerminalApplicationPlatform = "ios"
```

Comment:
```suggestion
	TerminalApplicationPlatform_iOS     TerminalApplicationPlatform = "ios"
```
Do we have to capitalize after the underscore?